test/spec/spectest.cpp

```diff
@@ -229,6 +230,7 @@ static const TestsuiteProposal TestsuiteProposals[] = {
     {"tail-call"sv, {Proposal::TailCall}},
     {"extended-const"sv, {Proposal::ExtendedConst}},
     {"threads"sv, {Proposal::Threads}},
+    {"exception-handling"sv, {Proposal::ExceptionHandling, Proposal::TailCall}},
```

Comment:
@harry900831 
After the PR #2801 being merged, I'll rebase the branch.You can add the third param `WasmEdge::SpecTest::TestMode::Interpreter` here. Then the CI will pass.

Reply:
Done